### PR TITLE
[Form] Show all option normalizers on debug:form command

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT

Follow-up https://github.com/symfony/symfony/pull/30371

normalizers

### CODE REVIEW

File: src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php [hunk 1]
```diff
@@ -164,27 +164,29 @@ public function testDebugCustomFormTypeOption()
...
+                   ]         %s
+ ---------------- -----------%s
+  Allowed values   [         %s
+                     "bar",  %s
+                     "baz"   %s
+                   ]         %s
+ ---------------- -----------%s
+  Normalizers      [         %s
+                     Closure(%s
+                       class:%s
```
Comment (xabbuh) [inline src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php]: This change looks a bit weird. Is this really necessary?

**Comment by OskarStark** [comment]
A bit of topic but as you are on it... In the Allowed types section shouldn’t it be null without quotes? 🤔

File: src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php]: It is hard have to update everytime these fixtures with details (width, spaces, endlines, etc) I think by shorting the output would avoid bother in the future.

Please, let me know if this does not make sense or if I'm missing something important to test hiding the details of the closure, which is unreadable IMO.

**Comment by yceruto** [comment]
Hi @OskarStark, the output is consistent with its definition, we pass `"null"` as allowed type.

**Comment by yceruto** [comment]
Rebased

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @yceruto.

**Pull request merged**